dashboard/modules/job/data_types.py

```diff
@@ -17,11 +17,24 @@ def __str__(self):
 # ==== Get Package ====
 
 
+@dataclass
+class GetPackageRequest:
+    package_uri: str
+
+
```

Comment:
don't think the changes to this file are necessary

Reply:
this is kinda overfitting to current approach to reuse request validation logic, but i agree we probably don't need to add request types for most of these for a long time ..